pkg/resources/stream_on_table.go

```diff
@@ -162,26 +162,24 @@ func CreateStreamOnTable(orReplace bool) schema.CreateContextFunc {
 		req := sdk.NewCreateOnTableStreamRequest(id, tableId)
 		if orReplace {
 			req.WithOrReplace(true)
+		} else if d.Get("copy_grants").(bool) {
+			return diag.Diagnostics{
+				diag.Diagnostic{
+					Severity: diag.Error,
+					Summary:  "COPY GRANTS cannot be used without OR REPLACE. If you are creating this object, first import it, so that it's managed by the provider, and try again.",
```

Comment:
I guess it's already in other thread, but I think we could have one pass in Create that doesn't task copy_grants into account, but later modifications will be counted. Importing to use copy_grants is cumbersome (I know I can set it to false and then to true when it's in state, but not everybody will understand that).

Reply:
You can also leave it empty, which I think most of the users will do. Besides, that's the default workflow here - when you have an existing object outside Terraform, you should import it first.

Reply:
Yeah, but it's not intuitive, because this parameter blocks Create operation suggesting import when it could be just created without that parameter and then set to true (if you know how the resource works underneath). Even if the resource already exists and someone has copy_grants set to true he'll get error from Snowflake saying 'object already exists', then the user may consider importing or removing the object manually and creating it through Terraform (with either copy_grants turned on or off without any blocking). That's how I see it.

Reply:
OK, changed it, but it's still not intuitive to me.